mymodule/mymodule.cpp

```diff
@@ -261,7 +261,18 @@ void sec_websocket_accept(const void * src, void * dst) {
 
 PyObject * c_accept(PyObject * self, PyObject * arg) {
     char result[28];
-    sec_websocket_accept(PyUnicode_AsUTF8(arg), result);
+    const char* src;
+    Py_ssize_t len;
+    src = PyUnicode_AsUTF8AndSize(arg, &len);
```

Comment:
before this `PyUnicode_CheckExact(arg)` is required.

Reply:
Not necessarily. It looks like an integer input will just cause a `TypeError: bad argument type for built-in operation` without it.